auth/src/commonMain/kotlin/com/liftric/auth/jwt/IdTokenSerializer.kt

```diff
@@ -103,6 +103,12 @@ internal object CustomAttributesSerializer : KSerializer<CognitoIdTokenClaims> {
         }?: run {
             throw SerializationException("Missing field event_id")
         }
+        val jti = filtersMap["jti"]?.let {
```

Comment:
Either
```
        val jti: String? = filtersMap["jti"]?.let {
            json.decodeFromJsonElement(it)
        }
```
or
```
        val jti = filtersMap["jti"]?.let {
            json.decodeFromJsonElement<String?>(it)
        }
```

would be more idiomatic compared to using the generated serializer directly